releasenotes/notes/solver-generalization-ad588a54c830c66f.yaml

```
@@ -0,0 +1,27 @@
+---
+upgrade:
+  - |
+    The :class:`~qiskit_dynamics.Solver` class has been modified to no longer
+    internally store signals: rather than passing them at instantiation, or setting them
+    via the ``signals`` property, ``signals`` must be passed as an argument to the
+    ``solve`` method when solving. Setting ``signals`` at instantiation or through the
+    property has been deprecated (see deprecations). A new instantiation argument,
+    ``rwa_carrier_freqs``, has been added to control behaviour of the rotating wave
+    approximation in the absence of signals being specified.
+  - |
+    The ``solve`` method of :class:`~qiskit_dynamics.Solver` can now be made to perform
```

This bullet should go under `features` since its adding a new feature not a breaking API change

```suggestion
    The  :meth:`.Solver.solve` method can now run multiple simulations in a single call by specifying...
```

Might be worth noting that these simulations are not currently parallelized are are run one after the other?

---

Moved this to `features`, and also added an extra sentence to specify that the simulations are done in a serial loop.